### PR TITLE
AISDK-229: Add skip_postprocessing option

### PR DESCRIPTION
**Pull request by kbridbur**

### CODE REVIEW

File: src/test/java/ai/rev/speechtotext/unit/RevAiStreamingClientOptionalParametersTest.java [hunk 1]
```diff
@@ -48,17 +48,17 @@ public static String[] vocabIdAndNull() {
...
   @DataPoints("deleteAfterSecondsAndNull")
   public static Integer[] deleteAfterSecondsAndNull() {
-    return new Integer[] {0, 1, null};
+    return new Integer[] {1, null};
   }
 
   @DataPoints("booleanValuesAndNull")
   public static Boolean[] booleanValuesAndNull() {
-    return new Boolean[] {true, false, null};
+    return new Boolean[] {false, null};
```
Comment (beaudrychase) [inline src/test/java/ai/rev/speechtotext/unit/RevAiStreamingClientOptionalParametersTest.java:56]: I'm a little confused by this change. Why isn't true a value here

**Review by beaudrychase — APPROVED** [review]
I had one question but it looks good to me

File: src/test/java/ai/rev/speechtotext/unit/RevAiStreamingClientOptionalParametersTest.java (hunk 1 above)
Comment (kbridbur) [reply src/test/java/ai/rev/speechtotext/unit/RevAiStreamingClientOptionalParametersTest.java:56]: This test had too many possibilities (testing machine was running out of memory) so i got rid of unnecessary options to reduce it. the purpose of the test is to make sure null values don't appear in the query string, so testing anything other than 1 null and 1 nonnull is unneeded

Comment (beaudrychase) [reply src/test/java/ai/rev/speechtotext/unit/RevAiStreamingClientOptionalParametersTest.java:56]: makes sense

**Pull request merged**